client/lib/signup/site-type.js

```diff
@@ -0,0 +1,59 @@
+/** @format **/
+/**
+ * Exernal dependencies
+ */
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+
+/**
+ * Current list of verticals that have/will have landing pages.
+ * A user who comes in via a landing page will not see the Site Topic dropdown.
+ */
+export const allSiteTypes = [
+	{
+		type: i18n.translate( 'blogger' ),
+		description: i18n.translate( 'Share a collection of posts.' ),
+	},
+	{
+		type: i18n.translate( 'business' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **61** times:
`translate( 'Business' )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-signup-site-type-step)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).